sites/dwhdelft.nl/components/JoinOptions.vue

```diff
@@ -37,8 +43,16 @@ nl:
       <div class="flex-1 mx-2 flex items-end mt-4">
         <div class="text-white">
           <h1 class="font-medium leading-none text-5xl mb-4" v-html="$t('title')"></h1>
-          <p class="text-lg lg:pr-40 mb-4" v-html="$t('description')" />
-          <p class="text-lg lg:pr-40" v-html="$t('outsite_hint')" />
+          <p class="text-lg lg:pr-20 mb-4" v-html="$t('description')" />
+          <p class="text-lg lg:pr-20 mb-6" v-html="$t('join')" />
```

Comment:
```suggestion
          <p class="text-lg lg:pr-20 mb-4" v-html="$t('join')" />
```
voor consistente spacing

Als alternatief kun je ook alle `mb-4`tje op deze elementen weglaten en op het divje erboven een `space-y-4` toevoegen

Reply:
Deze staat bewust anders, de spacing mb-4 is hetzelfde als boven met de intro paragrafen, de spacing mb-6 is consistent met de afstand die op outsite.nl tussen tekst en button zit. Omdat zo'n knop best bulky is aan de bovenkant vergeleken met tekst, is de afstand dan voor het oog juist gelijk.

Zie trouwens wel dat bij barbuddies het weer mb-8 is, dus so much for consistent zijn 😅